gen_cfg_section.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import os
+import re
+from urllib.request import urlopen
+
+import yaml
+
+
+def parse_cli_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        'api_versions',
+        nargs='+',
+        help=(
+            'Docker API version(s) to support. ' +
+            'If exactly two versions are passed, ' +
+            'it will act as a range.'
+        )
+    )
+    parser.add_argument(
+        '--yaml-dir',
```

Comment:
Not needed. Swagger YAML files can be in a directory you want. Just document that in the README and let users mount whatever they want in that volume.

Reply:
You mention elsewhere that you would like the YAML files to be downloaded and added to the Docker image via the Dockerfile. It seems like the method you proposed above is unnecessary if we add all of the YAML files directly to the image in such a fashion.

Reply:
Yes, it wouldn't be needed. I just  imagined that you personally wanted to do it for some reason, given you took the time to add an option for that, but you're right that this is not very important when the other changes are done.